src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String randomColor = colorSupplier.getRandomColor();
+        int randomNumber = new Random().nextInt(100) + 1; // 1 to 100
```

Comment:
ColorSupplier and Random should be private class fields, 100 - private class constant